lua/lspconfig/server_configurations/cairo_ls.lua

```diff
@@ -27,7 +24,7 @@ require'lspconfig'.cairo_ls.setup{}
 *cairo-language-server is still under active development, some features might not work yet !*
 ]],
     default_config = {
-      root_dir = [[root_pattern("package.json", "tsconfig.json", "jsconfig.json", ".git")]],
+      root_dir = [[root_pattern("Scarb.toml", "cairo_project.toml", ".git")]],
```

Comment:
where the "package.json", "tsconfig.json" entries accidental?

Reply:
I assume so. The docs didn't match the `root_dir` field above so I just updated the two to match whilst I was in there.